### ISSUE TITLE
get_synthesis_plan errors out when `actions` are None

### ISSUE THREAD

**Issue by rawkintrevo** [rxn4chemistry/rxn4chemistry#54]
When trying to retrieve synthesis actions, I recieved an unclear error:

```
Traceback (most recent call last):
  File "/Users/rawkintrevo/Library/Application Support/JetBrains/IntelliJIdea2022.3/plugins/python/helpers/pydev/pydevconsole.py", line 364, in runcode
    coro = func()
  File "<input>", line 5, in <module>
  File "/Users/rawkintrevo/miniforge3/envs/scidk/lib/python3.8/site-packages/rxn4chemistry/core.py", line 956, in get_synthesis_plan
    flattened_actions.extend(node["actions"])
TypeError: 'NoneType' object is not iterable
```

After digging in, we found that `node['actions']` is a `None` type, hence the error. 

It is unclear why `node['actions']` is `None` but there should be something that happens somewhere to prevent from getting to that error state or to gracefully handle the error when that state is achieved.

**Comment by drugilsberg** [comment]
@rawkintrevo small update, this PR (#55) should resolve the issues by properly deprecating some endpoints that are not needed anymore.

**Comment by drugilsberg** [comment]
closing since resolved in #55 with the 1.6.4 release.
